mosad_backend/pom.xml

```diff
@@ -53,28 +53,41 @@
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
 		</dependency>
+<<<<<<< HEAD
+=======
 		<dependency>
 			<groupId>org.modelmapper</groupId>
 			<artifactId>modelmapper</artifactId>
 			<version>3.2.2</version>
 		</dependency>
+>>>>>>> 8ad0735d9b9dd25292ea1444b9e2b50deaa33484
 
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
 		<dependency>
-			<groupId>org.springframework.security</groupId>
-			<artifactId>spring-security-test</artifactId>
-			<scope>test</scope>
+			<groupId>org.modelmapper</groupId>
```

Comment:
There is another modelMapper dependency. in pom xml. This is not needed. CHECK THAT AGAIN